cluster/evaluator/launch_eval.py

```diff
@@ -49,27 +52,19 @@ def launch_eval_job(m1_path, m2_path, job_name, bucket_name, completions=5):
 
     raw_job_conf = open("cluster/evaluator/cc-evaluator.yaml").read()
 
+    if flags_path:
+        os.environ['EVAL_FLAGS_PATH'] = flags_path
     os.environ['BUCKET_NAME'] = bucket_name
-
     os.environ['MODEL_BLACK'] = m1_path
     os.environ['MODEL_WHITE'] = m2_path
-    os.environ['JOBNAME'] = job_name + '-bw'
+    os.environ['JOBNAME'] = job_name
     env_job_conf = os.path.expandvars(raw_job_conf)
 
     job_conf = yaml.load(env_job_conf)
     job_conf['spec']['completions'] = completions
 
-    resp_bw = api_instance.create_namespaced_job('default', body=job_conf)
-
-    os.environ['MODEL_WHITE'] = m1_path
-    os.environ['MODEL_BLACK'] = m2_path
-    os.environ['JOBNAME'] = job_name + '-wb'
-    env_job_conf = os.path.expandvars(raw_job_conf)
-    job_conf = yaml.load(env_job_conf)
-    job_conf['spec']['completions'] = completions
-
-    resp_wb = api_instance.create_namespaced_job('default', body=job_conf)
```

Comment:
where did this go?

Reply:
default params are parallel_games=2 now :)